### ISSUE TITLE
Received [Empty Slot] Spellbook from Quest Rewards

### ISSUE THREAD

**Issue by CaldwellYSR** [adam9899/Issues#248]
I just completed the Wizard Armor quest and the random spellbook reward from it gave me a spell book - [Empty Slot].

**Comment by ByteTemplar** [comment]
Is your world from before 1.3.1 time? That removed one of Electrobobs addon mods, that added more spells and I think those books might be messing with people, that played before removal.

**Comment by CaldwellYSR** [comment]
I don't think so. 1.3.2 and now 1.3.3

**Comment by CaldwellYSR** [comment]
Yeah @ByteTemplar my folder is named Eternal 1.3.2 so it must have been something that got removed with the 1.3.3 update.

**Comment by JohnTitor95** [comment]
Actually, i've only ever played on 1.3.3 version (installed as 1.3.3 to begin with) and i still get those. Not sure if they were from quest rewards or random loot though.
